README.md

```diff
@@ -136,6 +136,21 @@ from the classpath are loaded. Next, if `iceberg.hadoop-conf-dir` is specified,
 are loaded from that location. Finally, any `iceberg.hadoop.*` properties from the sink config are
 applied. When merging these, the order of precedence is sink config > config dir > classpath.
 
+## GCS configuration
```

Comment:
We should create a section under `Catalog configuration` for the three main file IOs - S3FileIO, GCSFileIO, and ADLSFileIO. It doesn't make sense to have a special section for GCS without the others. Also, it should just be an example as more detailed docs belong in the Iceberg docs, otherwise we will end up duplicating all Iceberg catalog config docs here.

I don't believe there are Iceberg docs for GCSFileIO or ADLSFileIO, so contributing those to the Iceberg repo would be appreciated by the community I'm sure.